source/reference/glossary.md

```diff
@@ -25,6 +25,10 @@ NixOS
 
     Read /nɪks oʊ ɛs/ ("Niks Oh Es").
 
+Nix platform
+    The set of components that combine to create the official Nix ecosystem:
+    Nix, Nixpkgs, NixOS.
+
```

Comment:
```suggestion
Nix platform
    The officially supported software components that form the basis of the {term}`Nix ecosystem`:
    Nix, Nixpkgs, NixOS.

Nix ecosystem
    Products and services based on the {term}`Nix platform`.

```

Should also define Nix ecosystem there while we're at it.

Reply:
```suggestion
Nix platform
    The officially supported software components that form the basis of the {term}`Nix ecosystem`:
    Nix, Nixpkgs, NixOS.

Nix ecosystem
    All products and services based on the {term}`Nix platform`.

```

I think we should keep "the" as they are definite. How about "all" for the ecosystem?